src/Aspire.Hosting/Dcp/ResourceLogSource.cs

```diff
@@ -25,6 +25,8 @@ public async IAsyncEnumerator<LogEntryList> GetAsyncEnumerator(CancellationToken
         }
 
         var timestamps = resource is Container; // Timestamps are available only for Containers as of Aspire P5.
+        var startupStdoutStream = await kubernetesService.GetLogStreamAsync(resource, Logs.StreamTypeStartupStdOut, follow: true, timestamps: timestamps, cancellationToken).ConfigureAwait(false);
```

Comment:
There are no startup logs for Executables at the moment. Asking for them (for Executables) just slows things down a bit.

Reply:
Fixed